### ISSUE TITLE
attribute description with value NA causes java exception

### ISSUE THREAD

**Issue by hjvdwijk** [obiba/opal#3606]
**Describe the bug**
attribute description with value NA (type char) causes java exception
Error: Client error: (400) Bad Request; MagmaRuntimeException: java.lang.NullPointerException

**To Reproduce**

```
test_data <- tibble(
  id = c(1,2,3),
  sex = c("M","F","M")
)
attr(test_data$sex , "description") <- as.character(NA)
o <- opal.login(token = keyring::key_get(config$opal$keyring$token_name), url = config$opal$url)
opal.table_save( opal = o, 
                 project = project_name,
                 tibble = test_data, 
                 table = 'test_data', 
                 overwrite = TRUE, 
                 force = TRUE, 
                 type = entity_type
)
```

**Comment by ymarcon** [comment]
java stack trace:

```
Caused by: java.lang.NullPointerException: null
	at org.obiba.opal.spi.r.datasource.magma.RVariableValueSource.extractLocalizedAttributes(RVariableValueSource.java:238) ~[opal-spi-r-3.1-SNAPSHOT.jar:na]
	at org.obiba.opal.spi.r.datasource.magma.RVariableValueSource.extractAttributes(RVariableValueSource.java:221) ~[opal-spi-r-3.1-SNAPSHOT.jar:na]
	at org.obiba.opal.spi.r.datasource.magma.RVariableValueSource.initialiseVariable(RVariableValueSource.java:119) ~[opal-spi-r-3.1-SNAPSHOT.jar:na]
	at org.obiba.opal.spi.r.datasource.magma.RVariableValueSource.<init>(RVariableValueSource.java:58) ~[opal-spi-r-3.1-SNAPSHOT.jar:na]
	at org.obiba.opal.spi.r.datasource.magma.RValueTable.initialiseVariables(RValueTable.java:106) ~[opal-spi-r-3.1-SNAPSHOT.jar:na]
	at org.obiba.opal.spi.r.datasource.magma.RValueTable.initialise(RValueTable.java:51) ~[opal-spi-r-3.1-SNAPSHOT.jar:na]
	at org.obiba.magma.support.Initialisables.initialise(Initialisables.java:24) ~[magma-api-2.1.2.jar:2.1.2]
	... 74 common frames omitted
```

**Comment by ymarcon** [comment]
Fix available in opal 3.0.8
